src/build.sh

```diff
@@ -31,6 +31,7 @@ function build_kernel_main()
   local llvm
   local env_name
   local clean
+  local flag
```

Comment:
This variable has already been declared.

Reply:
ok, i'm fixing this.